lib/expression/v2/compile.ex

```diff
@@ -107,39 +120,17 @@ defmodule Expression.V2.Compile do
 
   defp quoted({"__property__", [a, b]}) when is_binary(b) do
     # When the property we're trying to read is a binary then we're doing
-    # `foo.bar` in an expression and we convert this to a `Map.get(foo, "bar")`
-    {{:., [], [{:__aliases__, [alias: false], [:Map]}, :get]}, [],
-     [
-       quoted(a),
-       b
-     ]}
-  end
-
-  defp quoted({"__attribute__", [a, b]}) when is_integer(b) do
-    # When the attribute we're trying to read is an integer then we're
-    # trying to read an index off of a list.
-    # `foo[1]` becomes `Enum.at(foo, 1)`
-    {{:., [], [Enum, :at]}, [],
-     [
-       quoted(a),
-       quoted(b)
-     ]}
```

Comment:
`foo["bar"]` for properties ended up being a lot more concise and readable than a lot of nested Map.get calls.